server/controllers/list.js

```diff
@@ -35,18 +35,24 @@ const listClients = require('../sockets/index').getListViewClients();
 const socketInstance = require('../sockets/index').getSocketInstance();
 const { createListCohort } = require('../sockets/cohort');
 const {
-  addListViewer,
-  addMemberRoleInList,
-  addOwnerRoleInList,
-  archiveList,
-  changeListType,
-  deleteList,
-  leaveList: leaveListSocket,
-  removeListMember,
-  removeMemberRoleInList,
-  removeOwnerRoleInList,
-  restoreList,
-  updateList
+  addItemToList: addItemWS,
+  addListViewer: addListViewerWS,
+  addMemberRoleInList: addMemberRoleInListWS,
+  addOwnerRoleInList: addOwnerRoleInListWS,
+  archiveList: archiveListWS,
+  changeListType: changeListTypeWS,
+  clearVote: clearVoteWS,
+  cloneItem: cloneItemWS,
+  deleteItem: deleteItemWS,
+  deleteList: deleteListWS,
+  leaveList: leaveListWS,
+  removeListMember: removeListMemberWS,
+  removeMemberRoleInList: removeMemberRoleInListWS,
+  removeOwnerRoleInList: removeOwnerRoleInListWS,
+  restoreList: restoreListWS,
+  setVote: setVoteWS,
+  updateItem: updateItemWS,
+  updateList: updateListWS
 } = require('../sockets/list');
```

Comment:
Consider using namespace vs the postfixes. For example `const socketActions = require('../sockets/list')` and then instead of `addItemWS` please use `socketActions.addItem`.